languages/zh-CN/README.md

```diff
@@ -0,0 +1,142 @@
+# CloudEvents 规范
```

Comment:
Should this be under the "cloudevents" dir? Or should there be a "cloudevents" subdir under "languages" ?
I'm leaning more towards "cloudevents/languages/..."